src/utils/http/cdfHttpClient.ts

```diff
@@ -125,6 +138,15 @@ export class CDFHttpClient extends RetryableHttpClient {
     }
   }
 
+  private enrichWithOneTimeHeaders(headers?: HttpHeaders) {
```

Comment:
Great name!

Reply:
made 3 different options for you to pick the best :D